install/kubernetes/cilium/charts/config/values.yaml

```diff
@@ -47,3 +47,6 @@
 # decisions can be checked via the poicy verdict messages.
 #policyAuditMode: false
 
+# enables passing identity on local routes by using the mark fields. However,
+# in cases where this conflicts with a chained CNI plugin it may be disabled.
+#enableIdentityMark: true
```

Comment:
I don't see the corresponding chaining docs changes for this, did you intend to add them here?

Reply:
We should set it in the Calico guide